### ISSUE TITLE
Automatic refresh of newly connected devices

### ISSUE THREAD

**Issue by Lyton505** [lakinduakash/linux-wifi-hotspot#264]
I notice how you have to hit refresh in order to see which devices have connected or disconnected, and I see it as a small inconvenience. To add to this already incredible project, I think the ability to automatically refresh would be fantastic.

**Comment by sigmaSd** [comment]
Good idea, it shouldn't be hard to do, I think gtk has something equivalent to js setInterval.
